Pod/Classes/ProductRecommendationModel.swift

```diff
@@ -81,6 +82,7 @@ public class ProductRecommendationModel: NSObject, NSCopying {
         self.currency = currency
         self.specialPrice = specialPrice
         self.specialPriceText = specialPriceText
+        self.region = region
```

Comment:
[Product object](https://github.com/segmentify/segmentify-v3/blob/dev/commons/src/main/java/com/segmentify/lotr/commons/model/product/Product.java) does not have region field. I think we do not need this change.